src/Avalonia.Styling/StyledElement.cs

```diff
@@ -235,6 +236,7 @@ public Styles Styles
                     }
 
                     _styles.ResourcesChanged += ThisResourcesChanged;
+                    NotifyResourcesChanged(new ResourcesChangedEventArgs());
```

Comment:
What do you think about caching args instance in a static and reusing that? Would save us several thousands allocations when starting the app and loading more complex controls.

Reply:
Yeah we should do that. My vague plan was to indicate which resources changed in the event args, which would mean we can't do that, but for now it's worth doing.